packages/cozy-harvest-lib/src/helpers/konnectors.js

```diff
@@ -76,6 +76,11 @@ export class KonnectorJobError extends Error {
      */
     const type = this.code.split('.')[0]
     this.type = KNOWN_ERRORS.includes(type) ? type : UNKNOWN_ERROR
+
+    // I don't know where this is used, but in biWebview.js or budget-insight.js
+    // we assign the account id to the error. Should we remove this definition?
+    // or should we remove the assignation?
```

Comment:
This accountId is actually not used. I just added this as a debugging information. We can remove it here and in budget-insight.js and biWebview.js